[android][editor] 'No suitable category' info (#7748)
organicmaps/organicmaps: data/strings/strings.txt

    ar = لا يمكن العثور على فئة مناسبة؟
    az = Uyğun kateqoriya tapa bilmirsiniz?
    be = Не можаце знайсці прыдатную катэгорыю?
    bg = Не можете да намерите подходяща категория?
    ca = No trobeu una categoria adequada?
    cs = Nemůžete najít vhodnou kategorii?
    da = Kan du ikke finde en passende kategori?
    de = Keine passende Kategorie gefunden?
    el = Δεν μπορείτε να βρείτε την κατάλληλη κατηγορία;
    es = ¿No encuentras una categoría adecuada?
    et = Ei leia sobivat kategooriat?
    eu = Ezin duzu kategoria egokirik aurkitu?
    fa = آیا نمی توانید یک دسته بندی مناسب پیدا کنید؟
    fi = Etkö löydä sopivaa luokkaa?
    fr = Tu ne trouves pas de catégorie appropriée ?
    he = לא מוצאים קטגוריה מתאימה?
    hi = कोई उपयुक्त श्रेणी नहीं मिल रही?
    hu = Nem talál megfelelő kategóriát?
    id = Tidak dapat menemukan kategori yang sesuai?
    it = Non riesci a trovare una categoria adatta?
    ja = 適切なカテゴリーが見つからない？
    ko = 적합한 카테고리를 찾을 수 없나요?
    lt = Nerandate tinkamos kategorijos?
    mr = योग्य श्रेणी शोधू शकत नाही?
    nb = Finner du ikke en passende kategori?
    nl = Kun je geen geschikte categorie vinden?
    pl = Nie możesz znaleźć odpowiedniej kategorii?

>     pt = Não consegues encontrar uma categoria adequada?
>     pt-BR = Você não consegue encontrar uma categoria adequada?

Reviewer: ```suggestion
    pt = Não encontra uma categoria adequada?
    pt-BR = Você não encontra uma categoria adequada?
```

    ro = Nu găsiți o categorie potrivită?
    ru = Не можешь найти подходящую категорию?
    sk = Nemôžete nájsť vhodnú kategóriu?
    sv = Kan du inte hitta en lämplig kategori?
    sw = Je, huwezi kupata aina inayofaa?
    th = ไม่พบหมวดหมู่ที่เหมาะสมใช่ไหม?
    tr = Uygun bir kategori bulamıyor musunuz?
    uk = Не можете знайти відповідну категорію?
    vi = Không thể tìm thấy một danh mục phù hợp?
    zh-Hans = 找不到合适的类别？
    zh-Hant = 找不到合適的類別？

  [editor_category_unsuitable_text1]
    tags = android

>     en = Organic Maps does not support all categories. E.g. cities, railway stations, roads, lakes, or buildings can not be added. Please add such features directly via osm.org (use the Edit button in the top bar) or use a specialized OSM editing app.

Reviewer 2: Maybe first sentence like
`Organic Maps allows to add simple point features only, that means no towns, roads, lakes, building outlines, etc.`
?

> Please add such features directly via osm.org (use the Edit button in the top bar) or use 

Here I read it like there is an Edit button in OM's top bar that'll get me to the right place in osm.org... Maybe drop the `(use the Edit button in the top bar)` completely? or rephrase

Reviewer 3: "objects" then is clearer for newbies than a highly specific term "features".

Reviewer 2: "objects", "map objects", "object types", etc. should be good indeed

Reviewer 4: I second the suggestion for the first sentence, clarifying that only points are supported. Here's my slight edit to the suggestion:

```suggestion
    en = Organic Maps can only add point features, not lines or shapes. This means you cannot add roads, buildings, cities, railway stations, or lakes, among other things. Please add such features directly via osm.org (use the Edit button in the top bar of the website) or use a specialized OSM editing app.
```

I've also made roads and buildings first in the list of examples (they get the point across better than cities IMO) and clarified the top bar as noted.

    fa = نقشه های ارگانیک از همه دسته ها پشتیبانی نمی کند. به عنوان مثال. شهرها، ایستگاه های راه آهن، جاده ها، دریاچه ها یا ساختمان ها را نمی توان اضافه کرد. لطفاً چنین ویژگی هایی را مستقیماً از طریق osm.org اضافه کنید (از دکمه ویرایش در نوار بالا استفاده کنید) یا از یک برنامه ویرایش تخصصی OSM استفاده کنید.
    fi = Organic Maps ei tue kaikkia luokkia. Esimerkiksi kaupunkeja, rautatieasemia, teitä, järviä tai rakennuksia ei voi lisätä. Lisää tällaiset ominaisuudet suoraan osm.org-sivuston kautta (käytä yläpalkin Muokkaa-painiketta) tai käytä erikoistunutta OSM-editointisovellusta.
    fr = Organic Maps ne prend pas en charge toutes les catégories. Par exemple, les villes, les gares, les routes, les lacs ou les bâtiments ne peuvent pas être ajoutés. Ajoute ces éléments directement via osm.org (utilise le bouton Modifier dans la barre supérieure) ou utilise une application d'édition OSM spécialisée.

>     he = מפות אורגניות לא תומכות בכל הקטגוריות. לְמָשָׁל. לא ניתן להוסיף ערים, תחנות רכבת, כבישים, אגמים או מבנים. אנא הוסף תכונות כאלה ישירות דרך osm.org (השתמש בכפתור העריכה בסרגל העליון) או השתמש באפליקציית עריכה מיוחדת של OSM.

Reviewer 4: ```suggestion
    he = האפליקציה Organic Maps יכולה להוסיף רק נקודות, לא קווים או צורות. למשל, לא ניתן להוסיף כבישים, מבנים, ערים, תחנות רכבת, או אגמים. ניתן להוסיף תכונות כאלה ישירות דרך osm.org (השתמשו בכפתור העריכה בסרגל העליון באתר) או דרך אפליקציית עריכה ייעודית של OSM.
```

I based this on the suggested rephrasing I posted above (which was based on pastk's suggestion) not the current commit. Here's a back-translation to English (via google translate):

> The Organic Maps app can only add points, not lines or shapes. For example, you cannot add roads, buildings, cities, train stations, or lakes. Such features can be added directly through osm.org (use the edit button in the top bar of the site) or through a dedicated OSM editor application.

    mr = ऑर्गेनिक नकाशे सर्व श्रेणींना समर्थन देत नाहीत. उदा. शहरे, रेल्वे स्थानके, रस्ते, तलाव किंवा इमारती जोडल्या जाऊ शकत नाहीत. कृपया अशी वैशिष्ट्ये थेट osm.org द्वारे जोडा (शीर्ष बारमधील संपादन बटण वापरा) किंवा विशेष OSM संपादन ॲप वापरा.
    nb = Organic Maps støtter ikke alle kategorier. Det er f.eks. ikke mulig å legge til byer, jernbanestasjoner, veier, innsjøer eller bygninger. Legg til slike funksjoner direkte via osm.org (bruk Rediger-knappen i topplinjen) eller bruk en spesialisert OSM-redigeringsapp.
    nl = Organic Maps ondersteunt niet alle categorieën. Je kunt bijvoorbeeld geen steden, stations, wegen, meren of gebouwen toevoegen. Voeg zulke functies rechtstreeks toe via osm.org (gebruik de knop Bewerken in de bovenste balk) of gebruik een gespecialiseerde OSM-bewerkingsapp.
    pl = Organic Maps nie obsługuje wszystkich kategorii. Na przykład nie można dodawać miast, stacji kolejowych, dróg, jezior lub budynków. Dodaj takie elementy bezpośrednio przez osm.org (użyj przycisku Edytuj na górnym pasku) lub skorzystaj ze specjalistycznej aplikacji do edycji OSM.

>     pt = Os Mapas Orgânicos não suportam todas as categorias. Por exemplo, não podes adicionar cidades, estações de comboio, estradas, lagos ou edifícios. Por favor, adiciona essas características diretamente através de osm.org (usa o botão Editar na barra superior) ou usa uma aplicação de edição OSM especializada.
>     pt-BR = O Organic Maps não é compatível com todas as categorias. Por exemplo, não é possível adicionar cidades, estações de trem, estradas, lagos ou edifícios. Adicione esses recursos diretamente em osm.org (use o botão Editar na barra superior) ou use um aplicativo de edição OSM especializado.

Reviewer: ```suggestion
    pt = O Organic Maps não suporta todas as categorias. Por exemplo, não é possível adicionar cidades, estações de comboio, estradas, lagos ou edifícios. Adicione esses elementos diretamente em osm.org (use o botão Editar na barra superior) ou use uma aplicação especializada em edição do OSM.
    pt-BR = O Organic Maps não suporta todas as categorias. Por exemplo, não é possível adicionar cidades, estações de trem, estradas, lagos ou edifícios. Adicione esses elementos diretamente em osm.org (use o botão Editar na barra superior) ou use um aplicativo especializado em edição do OSM.
```

    da = Alternativt kan du også bruge OSM Notes (osm.org/wiki/Notes) til rapporter til OSM-fællesskabet.
    de = Alternativ kannst du auch OSM Notes (osm.org/wiki/Notes) für Hinweise an die OSM Community verwenden.
    el = Εναλλακτικά, μπορείτε επίσης να χρησιμοποιήσετε το OSM Notes (osm.org/wiki/Notes) για αναφορές προς την κοινότητα του OSM.
    es = También puedes utilizar Notas OSM (osm.org/wiki/Notes) para informar a la Comunidad OSM.
    es-MX = Alternativamente, también puede utilizar Notas de OSM (osm.org/wiki/Notes) para informes a la Comunidad OSM.
    et = Alternatiivina võite kasutada OSMi märkmeid (osm.org/wiki/Notes) ka OSMi kogukonnale esitatavate aruannete jaoks.
    eu = Bestela, OSM Notes (osm.org/wiki/Notes) ere erabil dezakezu OSM Komunitateari txostenak egiteko.
    fa = همچنین میu200cتوانید از یادداشتu200cهای OSM (osm.org/wiki/Notes) برای گزارشu200cهایی به انجمن OSM استفاده کنید.
    fi = Vaihtoehtoisesti voit myös käyttää OSM Notesia (osm.org/wiki/Notes) OSM-yhteisölle tehtäviin raportteihin.
    fr = Tu peux aussi utiliser OSM Notes (osm.org/wiki/Notes) pour les rapports destinés à la communauté OSM.

>     he = לחלופין, אתה יכול גם להשתמש ב-OSM Notes (osm.org/wiki/Notes) לדיווחים לקהילת OSM.

Reviewer 4: Edit for `he`:
```suggestion
    he = לחלופין, ניתן להוסיף הערות למפה (osm.org/wiki/Notes) כדי לדווח לקהילת OSM.
```

Q: Does Organic Maps allow users to add notes to the map? If not I don't see the value of this string, or at least it needs to be clarified that OM doesn't support this either.

    nb = Organic Maps tillater kun å legge til enkle punktkategorier, det vil si ingen byer, veier, innsjøer, bygningsomriss osv. Vennligst legg til slike kategorier direkte til <a href="https://www.openstreetmap.org">OpenStreetMap.org</a>. Sjekk vår <a href="https://organicmaps.app/faq/editing/advanced-map-editing">guide</a> for detaljerte trinnvise instruksjoner.
    nl = Met Organic Maps kun je alleen eenvoudige puntcategorieën toevoegen, dus geen steden, wegen, meren, contouren van gebouwen, enzovoort. Voeg zulke categorieën direct toe aan <a href="https://www.openstreetmap.org">OpenStreetMap.org</a>. Bekijk onze <a href="https://organicmaps.app/faq/editing/advanced-map-editing">gids</a> voor gedetailleerde stap voor stap instructies.
    pl = Organic Maps pozwala na dodawanie tylko prostych kategorii punktów, co oznacza brak miast, dróg, jezior, obrysów budynków itp. Dodaj takie kategorie bezpośrednio do <a href="https://www.openstreetmap.org">OpenStreetMap.org</a>. Sprawdź nasz <a href="https://organicmaps.app/faq/editing/advanced-map-editing">przewodnik</a>, aby uzyskać szczegółowe instrukcje krok po kroku.

>     pt = Organic Maps permite-te adicionar apenas categorias de pontos simples, ou seja, nada de cidades, estradas, lagos, contornos de edifícios, etc. Por favor, adiciona essas categorias diretamente ao <a href="https://www.openstreetmap.org">OpenStreetMap.org</a>. Consulta o nosso <a href="https://organicmaps.app/faq/editing/advanced-map-editing">guia</a> para instruções detalhadas passo a passo.
>     pt-BR = O Organic Maps permite que você adicione apenas categorias de pontos simples, o que significa que não há cidades, estradas, lagos, contornos de edifícios, etc. Adicione essas categorias diretamente ao <a href="https://www.openstreetmap.org">OpenStreetMap.org</a>. Consulte nosso <a href="https://organicmaps.app/faq/editing/advanced-map-editing">guia</a> para obter instruções detalhadas passo a passo.

Reviewer: ```suggestion
    pt = O Organic Maps permite-lhe adicionar apenas categorias de pontos simples, ou seja, nada de cidades, estradas, lagos, contornos de edifícios etc. Por favor, adicione essas categorias diretamente ao <a href="https://www.openstreetmap.org">OpenStreetMap.org</a>. Consulte o nosso <a href="https://organicmaps.app/faq/editing/advanced-map-editing">guia</a> para instruções detalhadas passo a passo.
    pt-BR = O Organic Maps permite que você adicione apenas categorias de pontos simples, o que significa que não há cidades, estradas, lagos, contornos de edifícios etc. Por favor, adicione essas categorias diretamente no <a href="https://www.openstreetmap.org">OpenStreetMap.org</a>. Consulte nosso <a href="https://organicmaps.app/faq/editing/advanced-map-editing">guia</a> para obter instruções detalhadas passo a passo.
```

    fr = Tu ne trouves pas de catégorie appropriée ?
    he = לא מוצאים קטגוריה מתאימה?
    hi = कोई उपयुक्त श्रेणी नहीं मिल रही?
    hu = Nem találja a megfelelő kategóriát?
    id = Tidak dapat menemukan kategori yang sesuai?
    it = Non riesci a trovare una categoria adatta?
    ja = 適切なカテゴリーが見つからない？
    ko = 적합한 카테고리를 찾을 수 없나요?
    lt = Nerandate tinkamos kategorijos?
    mr = योग्य श्रेणी शोधू शकत नाही?
    nb = Finner du ikke en passende kategori?
    nl = Kun je geen geschikte categorie vinden?
    pl = Nie możesz znaleźć odpowiedniej kategorii?
    pt = Não encontra uma categoria adequada?
    pt-BR = Você não encontra uma categoria adequada?
    ro = Nu găsiți o categorie potrivită?
    ru = Не можешь найти подходящую категорию?
    sk = Nemôžete nájsť vhodnú kategóriu?
    sv = Kan du inte hitta en lämplig kategori?
    sw = Je, huwezi kupata aina inayofaa?
    th = ไม่พบหมวดหมู่ที่เหมาะสมใช่ไหม?
    tr = Uygun bir kategori bulamıyor musunuz?
    uk = Не можете знайти відповідну категорію?
    vi = Không thể tìm thấy một danh mục phù hợp?
    zh-Hans = 找不到合适的类别？
    zh-Hant = 找不到合適的類別？

  [editor_category_unsuitable_text]
    tags = android

>     en = Organic Maps allows to add simple point categories only, that means no towns, roads, lakes, building outlines, etc. Please add such categories directly to <a href="https://www.openstreetmap.org">OpenStreetMap.org</a>. Check our <a href="https://organicmaps.app/faq/editing/advanced-map-editing">guide</a> for detailed step by step instructions.

Reviewer 3: There is already "Categories?" mentioned in the title, no need to mention it here. We can actually include all 3 forms: categories + object types + features, bingo!
Also a few more improvements:

`en = Organic Maps allows adding only simple object types, which means no towns, roads, lakes, building outlines, etc. Please add or edit more complex features at <a href="https://www.openstreetmap.org/">OpenStreetMap.org</a>. Check our <a href="https://organicmaps.app/faq/editing/advanced-map-editing/">guide</a> for detailed step-by-step instructions and other recommended map editing apps.`

Author: > We can actually include all 3 forms: categories + object types + features, bingo!

Please not, that's just confusing

Author: For the future it would be good if you could at some point indicate that the English string is good and ready for translation reviews. It's not good that the translator now have to translate similar string over and over again.

Reviewer 3: Sorry, it is not enough time to manage that amount of PRs alone. En texts can be pre-approved before anything else next time, even in the chat.

Reviewer 2: another option is make separate translation PRs after the main one is merged (incl. en strings) - it'll reduce amount of non-relevant notifications for translators too

the downside is that its extra work and surely it'll get forgotten sometimes

Reviewer 2: (and with Weblate it'll be a non-issue)

    af = Organiese kaarte laat toe om slegs eenvoudige puntkategorieë by te voeg, dit beteken geen dorpe, paaie, mere, gebouomlynings, ens. Voeg asseblief sulke kategorieë direk by <a href="https://www.openstreetmap.org">OpenStreetMap.org< /a>. Gaan ons <a href="https://organicmaps.app/faq/editing/advanced-map-editing">gids</a> na vir gedetailleerde stap-vir-stap-instruksies.
    ar = تسمح الخرائط العضوية بإضافة فئات نقاط بسيطة فقط، وهذا يعني عدم وجود بلدات وطرق وبحيرات ومخططات للمباني وما إلى ذلك. يرجى إضافة هذه الفئات مباشرة إلى <a href="https://www.openstreetmap.org">OpenStreetMap.org</a>. راجع موقعنا <a href="https://organicmaps.app/faq/editing/advanced-map-editing">دليل</a> للحصول على تعليمات مفصلة خطوة بخطوة.
    az = Üzvi Xəritələr yalnız sadə nöqtə kateqoriyaları əlavə etməyə imkan verir, bu o deməkdir ki, şəhərlər, yollar, göllər, bina konturları və s. yoxdur. Lütfən, belə kateqoriyaları birbaşa <a href="https://www.openstreetmap.org">OpenStreetMap.org</a> saytına əlavə edin. Ətraflı addım-addım təlimatlar üçün <a href="https://organicmaps.app/faq/editing/advanced-map-editing">bələdçimizi</a> yoxlayın.

>     be = Organic Maps дазваляе дадаваць толькі простыя катэгорыі кропак, што азначае адсутнасць гарадоў, дарог, азёр, абрысаў будынкаў і г.д. Калі ласка, дадавайце такія катэгорыі непасрэдна на <a href="https://www.openstreetmap.org">OpenStreetMap.org</a>. Праверце наш <a href="https://organicmaps.app/faq/editing/advanced-map-editing">кіраўніцтва</a> для атрымання падрабязных пакрокавых інструкцый.

Reviewer 3: ```suggestion
    be = Organic Maps дазваляе дадаваць на мапу толькі простыя тыпы аб'ектаў, гэта значыць без гарадаў, дарог, азёр і будынкаў. Калі ласка, дадайце такія катэгорыі на сайце <a href="https://www.openstreetmap.org/">OpenStreetMap.org </a>. Мы таксама рэкамендуем азнаёміцца ​​з <a href="https://organicmaps.app/faq/editing/advanced-map-editing/">падрабязнымі інструкцыямі і іншымі прыкладаннямі для рэдагавання мапы</a>.
```

    fa = Organic Maps اجازه میu200cدهد فقط دستهu200cهای نقطهu200cای ساده را اضافه کند، یعنی هیچ شهر، جاده، دریاچه، طرح کلی ساختمان و غیره وجود ندارد. لطفاً این دستهu200cها را مستقیماً به <a href="https://www.openstreetmap.org">OpenStreetMap.org</a> اضافه کنید. برای دستورالعملu200cهای گام به گام، <a href="https://organicmaps.app/faq/editing/advanced-map-editing">راهنمای</a> ما را بررسی کنید.
    fi = Organic Maps mahdollistaa vain yksinkertaisten pisteluokkien lisäämisen, eli ei kaupunkeja, teitä, järviä, rakennusten ääriviivoja jne. Lisää tällaiset luokat suoraan <a href="https://www.openstreetmap.org">OpenStreetMap.org</a>. Tarkista <a href="https://organicmaps.app/faq/editing/advanced-map-editing">oppaastamme</a> yksityiskohtaiset ohjeet vaihe vaiheelta.
    fr = Organic Maps ne permet d'ajouter que des catégories de points simples, c'est-à-dire pas de villes, de routes, de lacs, de contours de bâtiments, etc. Merci d'ajouter ces catégories directement sur <a href="https://www.openstreetmap.org">OpenStreetMap.org</a>. Consulte notre <a href="https://organicmaps.app/faq/editing/advanced-map-editing">guide</a> pour obtenir des instructions détaillées étape par étape.

>     he = מפות אורגניות מאפשרות להוסיף קטגוריות נקודות פשוטות בלבד, כלומר ללא עיירות, כבישים, אגמים, קווי מתאר וכו'. אנא הוסף קטגוריות כאלה ישירות ל<a href="https://www.openstreetmap.org">OpenStreetMap.org</a>. עיין ב<a href="https://organicmaps.app/faq/editing/advanced-map-editing">מדריך</a> שלנו לקבלת הוראות צעד אחר צעד מפורטות.

Reviewer 4: ```suggestion
    he = באמצעות Organic Maps ניתן להוסיף קטגוריות נקודה פשוטות בלבד, כלומר לא ניתן להוסיף עיירות, כבישים, אגמים, מבנים וכו'. אנא הוסיפו קטגוריות כאלה ישירות ל-<a href="https://www.openstreetmap.org">OpenStreetMap.org</a>. עיינו <a href="https://organicmaps.app/faq/editing/advanced-map-editing">במדריך</a> שלנו לקבלת הוראות מפורטות.
```

    pt-BR = O Organic Maps permite que você adicione apenas categorias de pontos simples, o que significa que não há cidades, estradas, lagos, contornos de edifícios etc. Por favor, adicione essas categorias diretamente no <a href="https://www.openstreetmap.org">OpenStreetMap.org</a>. Consulte nosso <a href="https://organicmaps.app/faq/editing/advanced-map-editing">guia</a> para obter instruções detalhadas passo a passo.
    ro = Organic Maps permite adăugarea doar a unor categorii de puncte simple, ceea ce înseamnă că nu există orașe, drumuri, lacuri, contururi de clădiri, etc. Vă rugăm să adăugați astfel de categorii direct la <a href="https://www.openstreetmap.org">OpenStreetMap.org</a>. Consultați <a href="https://organicmaps.app/faq/editing/advanced-map-editing">ghidul nostru</a> pentru instrucțiuni detaliate pas cu pas.

>     ru = Organic Maps позволяет добавлять только простые категории точек, то есть никаких городов, дорог, озер, контуров зданий и так далее. Пожалуйста, добавляй такие категории непосредственно на <a href="https://www.openstreetmap.org">OpenStreetMap.org</a>. Ознакомься с нашим <a href="https://organicmaps.app/faq/editing/advanced-map-editing">руководством</a> для получения подробных пошаговых инструкций.

Reviewer 3: Organic Maps позволяет добавлять на карту только простые типы объектов, то есть никаких городов, дорог, озер, контуров зданий. Пожалуйста, добавляйте такие категории на сайте <a href="https://www.openstreetmap.org/">OpenStreetMap.org</a>. Так же рекомендуем ознакомиться с нашими <a href="https://organicmaps.app/faq/editing/advanced-map-editing/">подробными пошаговыми инструкциями и другими приложениями для редактирования карты</a>.

    sw = Ramani za Kikaboni huruhusu kuongeza kategoria rahisi pekee, hiyo inamaanisha hakuna miji, barabara, maziwa, muhtasari wa majengo, n.k. Tafadhali ongeza kategoria kama hizo moja kwa moja kwenye <a href="https://www.openstreetmap.org">OpenStreetMap.org</a>. Angalia <a href="https://organicmaps.app/faq/editing/advanced-map-editing">mwongozo wetu</a> kwa maagizo ya kina ya hatua kwa hatua.
    th = แผนที่ทั่วไปอนุญาตให้เพิ่มหมวดหมู่จุดธรรมดาเท่านั้น ซึ่งหมายความว่าไม่มีเมือง ถนน ทะเลสาบ โครงร่างอาคาร ฯลฯ โปรดเพิ่มหมวดหมู่ดังกล่าวลงใน <a href="https://www.openstreetmap.org">OpenStreetMap.org</a>. ดู<a href="https://organicmaps.app/faq/editing/advanced-map-editing">คำแนะนำ</a> ของเราเพื่อดูคำแนะนำโดยละเอียดทีละขั้นตอน
    tr = Organik Haritalar yalnızca basit nokta kategorileri eklemeye izin verir, yani kasaba, yol, göl, bina ana hatları vb. yoktur. Lütfen bu tür kategorileri doğrudan <a href="https://www.openstreetmap.org">OpenStreetMap.org</a>'a ekleyin. Adım adım ayrıntılı talimatlar için <a href="https://organicmaps.app/faq/editing/advanced-map-editing">rehberimize</a> göz atın.

>     uk = Organic Maps дозволяє додавати лише прості категорії точок, тобто без міст, доріг, озер, контурів будівель тощо. Будь ласка, додавайте такі категорії безпосередньо на <a href="https://www.openstreetmap.org">OpenStreetMap.org</a>. Перевірте наш <a href="https://organicmaps.app/faq/editing/advanced-map-editing">посібник</a> для отримання детальних покрокових інструкцій.

Reviewer 3: ```suggestion
    uk = Organic Maps дозволяють додавати до мапи лише прості типи об'єктів, тобто без міст, доріг, озер та контурів будівель. Будь ласка, додайте такі категорії на сайті <a href="https://www.openstreetmap.org/">OpenStreetMap.org</a>. Ми також рекомендуємо ознайомитися із <a href="https://organicmaps.app/faq/editing/advanced-map-editing/">детальними інструкціями та іншими додатками для редагування карти</a>.
```
